src/lib/components/chat/ChatWindow.svelte

```diff
@@ -15,24 +15,26 @@
 	import type { Model } from "$lib/types/Model";
 	import WebSearchToggle from "../WebSearchToggle.svelte";
 	import LoginModal from "../LoginModal.svelte";
-	import type { WebSearchUpdate } from "$lib/types/MessageUpdate";
+	import type { RAGUpdate } from "$lib/types/MessageUpdate";
 	import { page } from "$app/stores";
 	import DisclaimerModal from "../DisclaimerModal.svelte";
 	import FileDropzone from "./FileDropzone.svelte";
 	import RetryBtn from "../RetryBtn.svelte";
 	import UploadBtn from "../UploadBtn.svelte";
 	import file2base64 from "$lib/utils/file2base64";
 	import { useSettingsStore } from "$lib/stores/settings";
+	import type { PdfUploadStatus } from "$lib/types/PdfChat";
 
 	export let messages: Message[] = [];
 	export let loading = false;
 	export let pending = false;
 	export let shared = false;
 	export let currentModel: Model;
 	export let models: Model[];
-	export let webSearchMessages: WebSearchUpdate[] = [];
+	export let RAGMessages: RAGUpdate[] = [];
 	export let preprompt: string | undefined = undefined;
 	export let files: File[] = [];
+	export let uploadPdfStatus: PdfUploadStatus;
```

Comment:
```suggestion
	export let uploadPdfStatus: PdfUploadStatus | undefined = undefined;
```

fix the missing uploadPdfStatus in type checks